remote/terraform_state_file.go

```diff
@@ -46,7 +46,13 @@ func (state *TerraformState) IsRemote() bool {
 
 // Parse the Terraform .tfstate file from the location specified by workingDir. If no location is specified,
 // search the current directory. If the file doesn't exist at any of the default locations, return nil.
-func ParseTerraformStateFileFromLocation(workingDir string) (*TerraformState, error) {
+func ParseTerraformStateFileFromLocation(backend string, config map[string]interface{}, workingDir string) (*TerraformState, error) {
+	stateFile, ok := config["path"].(string)
+	if backend == "local" && ok {
+		if util.FileExists(stateFile) {
```

Comment:
Hm, not sure what the right behavior is here if the file doesn't exist. Should we just try to read it elsewhere? Return an empty `TerraformState`? Return an error?